sdk/client/api_authorization_resource.go

```diff
@@ -39,32 +39,15 @@ func (a *AuthorizationResourceApiService) GetPermissions(ctx context.Context, ty
 		returnValue interface{}
 	)
 
-	// create path and map variables
 	path := "/auth/authorization/{type}/{id}"
 	path = strings.Replace(path, "{"+"type"+"}", fmt.Sprintf("%v", type_), -1)
 	path = strings.Replace(path, "{"+"id"+"}", fmt.Sprintf("%v", id), -1)
 
 	headerParams := make(map[string]string)
+	headerParams["Accept"] = "application/json"
+
 	queryParams := url.Values{}
 	formParams := url.Values{}
-
-	// to determine the Content-Type header
-	contentTypes := []string{}
-
-	// set Content-Type header
-	contentType := selectHeaderContentType(contentTypes)
```

Comment:
This always return an empty `contentType`

```go
// to determine the Content-Type header
contentTypes := []string{}
// set Content-Type header
contentType := selectHeaderContentType(contentTypes)
if contentType != "" {
	headerParams["Content-Type"] = contentType
}
```

So it doesn't really make sense.

And this is just setting to `headerParams["Content-Type"] ="application/json"`

```go
// to determine the Content-Type header
contentTypes := []string{"application/json"}

// set Content-Type header
contentType := selectHeaderContentType(contentTypes)
if contentType != "" {
	headerParams["Content-Type"] = contentType
}
```

Reply:
And a similar thing happens with
```go
// to determine the Accept header
headerAccepts := []string{"application/json"}

// set Accept header
headerAccept := selectHeaderAccept(headerAccepts)
if headerAccept != "" {
	headerParams["Accept"] = headerAccept
}
```